src/fastlane/release_actions/lib/fastlane/plugin/release_actions/helper/commit/parser.rb

```diff
@@ -0,0 +1,128 @@
+require 'strscan'
+require_relative 'commit'
+
+class Commit
+  # Parser is a Conventional Commits 1.0.0 compliant parser that turns commit
+  # messages into structured data. It uses StringScanner from the stdlib to
+  # search through commit messages for tokens and extracts the type, scope,
+  # subject, body, and footer from a commit message. If the parser finds a
+  # commit that isn't written as a Conventional Commmit, it will default to
+  # the type "change" and extract the subject and body.
+  #
+  # For more details on the specification, see
+  # https://www.conventionalcommits.org/en/v1.0.0/#specification.
+  class Parser
```

Comment:
This is in a namespace so it's `Commit::Parser` -- does that help?

Reply:
A "commit" is just a VCS commit; ConventionalCommit is a particular format of that which you're trying to parse. But at this point I'm just bikeshedding, so we can drop it if you don't feel a rename adds useful context.

Reply:
Ah yeah - I hear ya. I'll leave it for now. My style would be to introduce specificity later if we had the need to parse other kinds of commits. Same with versions -- I just called it version since we're following only one particular format.